webpack.config.mjs

```diff
@@ -155,6 +158,11 @@ export default env => {
     ],
     resolve: {
       alias: extraResolveAliases,
+      // TODO remove after updating react:
+      fallback: {
+        'react/jsx-runtime': 'react/jsx-runtime.js',
+        'react/jsx-dev-runtime': 'react/jsx-dev-runtime.js',
+      },
```

Comment:
This is needed beacause React 17 does not have the proper ESM exports, but react-pdf imports `react/jsx-runtime` internally (they use the new-style jsx transform)